alpha/external-web/app/log4js.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * This file defines a configures the log4js logging library.
+ */
+var log4js = require('log4js')
+
+// TODO: Change the file path to a relative path.
+log4js.configure('/usr/src/app/app/config/log4js-configuration.json')
```

Comment:
This would work for the code above but we'd need some means to add that environment variable into the JSON configuration file that the library uses.